src/main/java/io/gravitee/policy/requestvalidation/ConstraintValidator.java

```diff
@@ -26,4 +26,7 @@
     boolean isValid(T value);
 
     String getMessageTemplate();
+
+    String getMessageRegexp();
```

Comment:
Not sure what is the purpose of such new method.
Seems to be used only by a single unit test (https://github.com/gravitee-io/gravitee-policy-request-validation/pull/11/files#diff-c644eb24ea414ca175d1dba9b5700c3eR298)

No way to avoid such addition just for testing purpose?

Reply:
Yes, you are right, it's useless method, it's just for test purpose. Fixed this.